manifest.yml

```
@@ -0,0 +1,6 @@
+---
+applications:
+-
+  buildpack:
```

Is the newline after `-` needed?
```suggestion
- buildpack:
```

---

not really, initially some syntax errors were causing weird issues so just followed as in example. Will remove it if not required.

---

fixed